distributed/tests/test_client.py

```diff
@@ -4095,6 +4095,7 @@ def run():
                     # This line runs only when all parties are inside the
                     # context manager
                     assert Client.current(allow_global=False) is c
+                    assert default_client() is c
```

Comment:
@fjetter does this test work for you?  I've extended the existing test for `Client.current` to also run the same test for `default_client`

Reply:
yes. That works